gammacat/info.py

```diff
@@ -30,6 +30,10 @@ def __init__(self):
         self.out_path = self.base_dir / 'output'
         self.webpage_path = self.base_dir / 'webpage'
 
+        # Input and output URL on github
```

Comment:
Line 33 - 36 might be not necessary -> see inline comment in gammacat/webpage.py